ledger/participant-state/kvutils/tools/README.md

```diff
@@ -1,37 +1,3 @@
-# `integrity-check`
```

Comment:
We might want to move this module to somewhere else later as this only contain the submission extractor (i.e., once we have moved the rest of the kvutils module out).

Reply:
I will take care of that.